executorlib/backend/cache_parallel.py

```
@@ -1,6 +1,7 @@
 import pickle
 import sys
 import time
+from typing import Any
```

Ruff (0.8.2)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)